.gitignore

```diff
@@ -104,7 +104,7 @@ venv.bak/
 .mypy_cache/
 
 # Ignore our customized config.json
-config.json
+*.json
```

Comment:
I don't think ignoring all jsons is a good idea in general. Could we just ignore config*.json and then people can name it like config_rockies.json or what have you?

Reply:
Good idea.  I'll just change this to config*.json.